egs/voxpopuli/ASR/local/compute_fbank.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python3
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
+#              2023  Brno University of Technology (authors: Karel Veselý)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""
+This file computes fbank features of VoxPopuli dataset.
+
+Usage example:
+
+  python3 ./local/compute_fbank.py \
+      --src-dir data/fbank --output-dir data/fbank \
+      --num-jobs 100 --num-workers 25 \
+      --prefix "voxpopuli-${task}-${lang}" \
+      --dataset train \
+      --trim-to-supervisions True \
+      --speed-perturb True
+
+It looks for raw CutSet in the directory data/fbank
+located at: `{src_dir}/{prefix}_cuts_{dataset}_raw.jsonl.gz`.
+
+The generated fbank features are saved in `data/fbank/{prefix}-{dataset}_feats`
+and CutSet manifest stored in `data/fbank/{prefix}_cuts_{dataset}.jsonl.gz`.
+
+The number of workers is smaller than nunber of jobs
+(see --num-jobs 100 --num-workers 25 in the example).
```

Comment:
Is it a hard constraint? If so, could you write a check in the code for it?

Reply:
it is not a hard constraint, there can be also less jobs  than workers, in this case some workers will be idle...

Reply:
ok, the assert is added into the code